### PR TITLE
Part 3 instructions 4192

### PR DESCRIPTION
**Pull request by imeecuison**
## Description
Part 3 of 4192 instructions

## Testing done
No testing done

## Screenshots
screen shot 2019-01-16 at 3 40 26 pm

## Acceptance criteria
- [ ]The content on the page titled For your employer or command matches the 4192 Content Document (per link in Supporting Artifacts above/screenshot below)

- [ ]The image displayed on this page is 5.2.6-image1 & 5.2.6-image2 (see Supporting Artifacts)
The content on the page titled Your Social Security number matches the 4192 Content Document (per link in Supporting Artifacts above/screenshot below)

- [ ] The image displayed on this page is 5.2.7-image (see Supporting Artifacts)
The content on the page titled Send the form to your past employer matches the 4192 Content Document (per link in Supporting Artifacts above/screenshot below)
Field validations (if applicable per the linked document for Validations above) are enforced in the UI
Error messages (if applicable per the linked document for Validations above) are displayed in the UI

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/content/instructionalPart3.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      Put the VA Claims Intake Center address listed below in{' '}
+      <strong>Box 2.</strong> This is where your employer will return the form
+      after they‘ve completed their sections.
+    </p>
+    <div className="blue-bar-block">
+      Department of Veterans Affairs Claims Intake Center PO Box 4444
+      Janesville, WI 53547-4444 Or fax them toll-free: 1-844-531-7818
+    </div>
+
+    <img src="/img/part3-image.png" alt="Box 1" />
```
Comment (aaronjlech) [inline src/applications/disability-benefits/all-claims/content/instructionalPart3.jsx]: should the alt be Box 2?

File: src/applications/disability-benefits/all-claims/content/instructionalPart3.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  <div>
+    <h4>Fill in return address</h4>
+    <p>
+      Put the VA Claims Intake Center address listed below in{' '}
+      <strong>Box 2.</strong> This is where your employer will return the form
+      after they‘ve completed their sections.
+    </p>
+    <div className="blue-bar-block">
+      Department of Veterans Affairs Claims Intake Center PO Box 4444
+      Janesville, WI 53547-4444 Or fax them toll-free: 1-844-531-7818
```
Comment (aaronjlech) [inline src/applications/disability-benefits/all-claims/content/instructionalPart3.jsx]: linkable phone number maybe?

**Review by aaronjlech — APPROVED** [review]
Looks good, just a few little things.

**Comment by zurbergram** [comment]
@department-of-veterans-affairs/code-reviewers-for-external-teams-group Could I get a code review on this at your earliest convenience. Thank you!

**Review by rianfowler — APPROVED** [review]

**Pull request merged**